app/webapp.py

```diff
@@ -31,8 +31,15 @@
 )
 from aiohttp.web_request import BaseRequest
 from aiohttp.web_response import StreamResponse
-from aioswitcher.api import Command, SwitcherApi
+from aioswitcher.api import Command, SwitcherType1Api, SwitcherType2Api
 from aioswitcher.schedule import Days
+from aioswitcher.device import (
+    DeviceState,
+    ThermostatFanLevel,
+    ThermostatMode,
+    ThermostatSwing,
+)
+from aioswitcher.api.remotes import SwitcherBreezeRemoteManager
```

Comment:
this will fails `isort`.
```suggestion
from aioswitcher.api.remotes import SwitcherBreezeRemoteManager
from aioswitcher.device import (
    DeviceState,
    ThermostatFanLevel,
    ThermostatMode,
    ThermostatSwing,
)
from aioswitcher.schedule import Days```